cluster.go

```diff
@@ -319,6 +319,11 @@ func (c *Cluster) Topo() ClusterTopo {
 func (c *Cluster) getTopo(p Client) (ClusterTopo, error) {
 	var tt ClusterTopo
 	err := p.Do(Cmd(&tt, "CLUSTER", "SLOTS"))
+	if tt == nil && err == nil {
+		//This will happen between when nodes starts coming up after cluster goes down and
+		//Cluster swarm yet not ready using those nodes.
+		err = errors.New("No Topology Available")
```

Comment:
I think a more specific error would be "no cluster slots assigned", what do you think?

Reply:
You are right. It's more relevant to put "```no cluster slots assigned```" instead of "```No Topology Available```" in this function.